WeScan/Scan/FocusRectangle.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FocusRectangle.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 16/11/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import UIKit
+
+final class FocusRectangle: UIView {
+    convenience init(touchPoint: CGPoint) {
+        let originalSize: CGFloat = 200
+        let finalSize: CGFloat = 80
+        
+        self.init(frame: CGRect(x: touchPoint.x - (originalSize / 2), y: touchPoint.y - (originalSize / 2), width: originalSize, height: originalSize))
+        
+        backgroundColor = .clear
+        layer.borderWidth = 2.0
+        layer.cornerRadius = 6.0
+        layer.borderColor = UIColor.yellow.cgColor
+        
```

Comment:
There are a lot of magic numbers and calculations heres, is there a way of breaking this down or putting some of these into variables to explain whats going on ?

Reply:
Resolved: 9d1e1e4